index.html

```diff
@@ -109,6 +109,40 @@ <h2>A project for first-time contributions</h2>
       <!-- make sure there is one line of space above and below your card -->
       <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+      <!-- ________ Debakalpa's card START ________  -->
+      <div class="card">
+        <p class="name">Debakalpa Roy</p>
+        <p class="contact">
+          <i class="fab fa-github"></i>
+          <a href="https://github.com/DivineRoot" target="_blank">DivineRoot</a>
+        </p>
+        <p class="about">
+          A CS Undergrad trying to learn about OSS ❤
+        </p>
+        <div class="resources">
+          <p>3 Useful Dev Resources</p>
+          <ul>
+            <li>
+
```

Comment:
Please remove this blank line

Reply:
I removed the line blank line at line number 126. Please have a look.